rct229/ruletest_engine/ruletest_jsons/system_types/System_9b_Warm_Air_Furnace_Gas.json

```diff
@@ -0,0 +1,78 @@
+{
+    "id": "ASHRAE229 1",
+    "ruleset_model_instances": [
+        {
+            "id": "RMD 1",
+            "buildings": [
+                {
+                    "id": "Building 1",
+                    "building_open_schedule": "Required Building Schedule 1",
+                    "building_segments": [
+                        {
+                            "id": "Building Segment 1",
+                            "zones": [
+                                {
+                                    "id": "Thermal Zone 1",
+                                    "thermostat_cooling_setpoint_schedule": "Required Cooling Schedule 1",
+                                    "thermostat_heating_setpoint_schedule": "Required Heating Schedule 1",
+                                    "terminals": [
```

Comment:
To me, it seems like [terminal heating_source](https://github.com/pnnl/ruleset-checking-tool/blob/develop/docs/ruleset_functions/baseline_systems/is_baseline_system_9b.md#:~:text=are_all_terminal_heat_sources_hot_water(B_RMR%2Cterminal_unit_id_list)%20%3D%3D%20TRUE) part is missing. Could you double-check this?